Ports/elfutils/package.sh

```diff
@@ -8,5 +8,36 @@ files="http://archive.ubuntu.com/ubuntu/pool/main/e/elfutils/elfutils_${version}
 auth_type='sha256'
 depends=('zlib' 'libfts' 'argp-standalone' 'musl-obstack' 'curl' 'libarchive' 'sqlite' 'gettext')
 configopts=(
-    '--disable-debuginfod'
+    "--disable-debuginfod"
+    "--enable-libdebuginfod=dummy"
+    "--disable-libdwfl"
+    "--enable-maintainer-mode"
 )
+
+pre_configure() {
+    # Rebuild after patching configure.ac to support serenity host.
+
+    # `automake` may exit with a warning about how there is a mismatch
+    # between the versions of autoconf and automake that were previously
+    # used to generate aclocal and specifed in configure.ac.
+    run automake --add-missing || true
+    run autoreconf
```

Comment:
The tarball arrives preconfigured, so we should just patch the existing `configure` file. Everything else risks many more compatibility issues.